src/pages/profilePage/index.tsx

```diff
@@ -27,43 +29,42 @@ import {
   ModalHeader,
   ModalOverlay,
 } from '@chakra-ui/modal'
+import Image from 'next/image'
 
-const mockData = {
-  name: 'Bob Tan',
-  gender: 'M',
-  batch: 'AY2022/23',
-  year: 2,
-  faculty: 'Computing',
-  major: 'Computer Science',
-  telegram: '@bobbytan',
-  discord: 'bobtan #123',
-  nus_email: 'bobtan@u.nus.edu',
-  personal_email: 'bobtan@gmail.com',
-  hobbies: ['fishing', 'coding', 'running'],
-  department: 'Software Development',
-  role: 'Backend Engineer',
-  projects: ['Atlas HRMS', 'DAO', 'Fintech Month'],
-}
-const ProfilePage = () => {
-  const query = trpc.useQuery(['member-profile.getMemberProfile', 'A0123456Z'])
-  if (!query.data) {
-    return <div>Loading...</div>
-  } else {
+const MOCK_STUDENT_ID = 'asd'
+const defaultImage = '/150.png'
+
+const ProfilePage = ({ studentId }: { studentId: string }) => {
+  const userQuery = trpc.useQuery(
+    ['member-profile.getMemberProfile', studentId],
+    {
+      refetchOnWindowFocus: false,
+    }
+  )
+  if (!userQuery || !userQuery.data) {
     return (
-      <Box className="flex flex-wrap justify-between gap-6 mt-4">
-        <ProfileInfo {...query.data} />
-        <Box className="flex flex-col">
-          <ProfilePicture />
-          <ProfileContactInfo {...query.data} />
-        </Box>
+      <Box className="flex justify-center">
+        <Spinner size="lg" />
       </Box>
     )
   }
+  if (!userQuery.data.user) {
+    return <p className={'text-3xl'}>Something is wrong</p>
```

Comment:
Can remove the {}

Reply:
1. I removed the {}
2. I believe it would be easier to just keep everything in one file to keep it simple as in the end, the popup will just be a button that is imported and not a page.

Reply:
If you are ok with the 2 things mentioned above, I think we can go ahead and merge the PR. Once ATL-3 is done, we can make another pull request to integrate ATL-4 and 3.